tests/topotests/bgp_addpath_best_selected/test_bgp_addpath_best_selected.py

```diff
@@ -111,6 +111,9 @@ def _bgp_converge():
     _, result = topotest.run_and_expect(test_func, None, count=30, wait=1)
     assert result is None, "Can't converge initially"
 
+    r2.vtysh_cmd("clear bgp ipv4 192.168.1.1 soft out")
```

Comment:
From my perspective, I would like the answer to whether or not this is a test failure or a bgp failure to be more explicit.  Issuing a clear can and will hide a bgp error itself